docs/setup/config.md

```diff
@@ -52,6 +52,14 @@ The configuration files for the TaskManagers can be different, Flink does not as
 - `parallelism.default`: The default parallelism to use for programs that have no parallelism specified. (DEFAULT: 1). For setups that have no concurrent jobs running, setting this value to NumTaskManagers * NumSlotsPerTaskManager will cause the system to use all available execution resources for the program's execution. **Note**: The default parallelism can be overwriten for an entire job by calling `setParallelism(int parallelism)` on the `ExecutionEnvironment` or by passing `-p <parallelism>` to the Flink Command-line frontend. It can be overwritten for single transformations by calling `setParallelism(int
 parallelism)` on an operator. See the [programming guide]({{site.baseurl}}/apis/programming_guide.html#parallel-execution) for more information about the parallelism.
 
+- `fs.default-scheme`: The default filesystem scheme to be used, with the necessary authority (if needed). 
+By default, this is set to `file:///` which points to the local filesystem. This means that the local 
```

Comment:
Actually it must be three. The authority in the case of the local filesystem is empty, this is denoted by having nothing between the two first and the third slashes.